exasol_integration_test_docker_environment/test/test_doctor.py

```diff
@@ -13,8 +14,8 @@
 )
 
 
-@contextmanager
-def temporary_env(env_vars) -> ContextManager[Mapping[str, str]]:
+@contextmanager #type: ignore
+def temporary_env(env_vars) -> Generator: #type: ignore
```

Comment:
Not sure how to get this one right.

Reply:
I think, we need to talk to @Nicoretti

Reply:
I think, we can now fix this, we now know how type context manager

Reply:
Fixed